docs/source/_templates/autosummary/class.rst

```diff
@@ -1,15 +1,15 @@
 {% extends "!autosummary/class.rst" %}
 
-{% set methods = methods|reject("equalto", "__init__") %}
+{% set methods =(methods| reject("equalto", "__init__") |list) %}
```

Comment:
Just curious what this `list` does

Reply:
This is because `| reject` creates a generator, which defies the purpose of `if methods` below, turning it back into a list will ensure that when the list is empty, we don't print the section header "Methods"